src/app/core/breadcrumbs/dso-name.service.ts

```diff
@@ -42,32 +42,33 @@ export class DSONameService {
         return `${firstName} ${lastName}`;
       }
     },
-    Person: (dso: DSpaceObject): string => {
-      const familyName = dso.firstMetadataValue('person.familyName');
-      const givenName = dso.firstMetadataValue('person.givenName');
+    Person: (dso: DSpaceObject, injectedAsHTML?: boolean): string => {
+      const familyName = dso.firstMetadataValue('person.familyName', undefined, injectedAsHTML);
+      const givenName = dso.firstMetadataValue('person.givenName', undefined, injectedAsHTML);
       if (isEmpty(familyName) && isEmpty(givenName)) {
-        return dso.firstMetadataValue('dc.title') || this.translateService.instant('dso.name.unnamed');
+        return dso.firstMetadataValue('dc.title', undefined, injectedAsHTML) || this.translateService.instant('dso.name.unnamed');
       } else if (isEmpty(familyName) || isEmpty(givenName)) {
         return familyName || givenName;
       } else {
         return `${familyName}, ${givenName}`;
       }
     },
-    OrgUnit: (dso: DSpaceObject): string => {
-      return dso.firstMetadataValue('organization.legalName') || this.translateService.instant('dso.name.untitled');
+    OrgUnit: (dso: DSpaceObject, injectedAsHTML?: boolean): string => {
+      return dso.firstMetadataValue('organization.legalName', undefined, injectedAsHTML);
     },
-    Default: (dso: DSpaceObject): string => {
+    Default: (dso: DSpaceObject, injectedAsHTML?: boolean): string => {
       // If object doesn't have dc.title metadata use name property
-      return dso.firstMetadataValue('dc.title') || dso.name || this.translateService.instant('dso.name.untitled');
+      return dso.firstMetadataValue('dc.title', undefined, injectedAsHTML) || dso.name || this.translateService.instant('dso.name.untitled');
     },
   };
 
   /**
    * Get the name for the given {@link DSpaceObject}
    *
    * @param dso  The {@link DSpaceObject} you want a name for
+   * @param injectedAsHTML Whether the HTML is used inside a `[innerHTML]` attribute
    */
-  getName(dso: DSpaceObject | undefined): string {
+  getName(dso: DSpaceObject | undefined, injectedAsHTML?: boolean): string {
```

Comment:
I find the variable name `injectedAsHTML` to be confusing.  I understand the intention is to pass `true` if the method is being called from an `[innerHTML]`.  But, doesn't this flag simply result in any included HTML being escaped? If so, I'd rather we call this `escapeHTML`, as that's describes the purpose of the flag better (rather than assuming it'll only ever be used with `[innerHTML]`).